src/core/eme/eme_manager.ts

```diff
@@ -229,7 +230,13 @@ export default function EMEManager(
         wantedSessionType = "persistent-license";
       }
 
-      return getSession(initializationData, stores, wantedSessionType)
+      const maxSessionCacheSize = typeof options.maxSessionCacheSize === "number" ?
+        options.maxSessionCacheSize :
+        EME_MAX_SIMULTANEOUS_MEDIA_KEY_SESSIONS;
```

Comment:
We had defined the value of EME_MAX_SIMULTANEOUS_MEDIA_KEY_SESSIONS to a maximum number of sessions we could create before the total amount of sessions takes too many resources.
Should we consider then that the given option may not be superior to this value? Isn't it too permissive to let the user set a too high value, considering moreover that the case where so many sessions are needed is an edge-case.

Reply:
I saw it as going both ways (lower than the default limit or higher than it) as a feature, for when you know your device and you know how many sessions it can handle.

What do you think about that?

But you're right that both the name of the config property and its documentation (thankfully it's not part of the API so we can change both) are wrong in that case and should be changed.

Reply:
Ok, I was then just wondering if it would break the API. But it seems not.
Ok for authorizing the user to set what he wants.

Reply:
I renamed it to `EME_DEFAULT_MAX_SIMULTANEOUS_MEDIA_KEY_SESSIONS`